Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -6102,7 +6126,7 @@ static BOOL shouldUseHighlightsForMarkedText(NSAttributedString *string)
         [hud mouseMoved:event];
 
     // When a view is first responder, it gets mouse moved events even when the mouse is outside its visible rect.
-    if (m_view.getAutoreleased() == [m_view window].firstResponder && !NSPointInRect([m_view convertPoint:[event locationInWindow] fromView:nil], [m_view visibleRect]))
+    if (m_view.getAutoreleased() == [m_view.get() window].firstResponder && !NSPointInRect([m_view.get() convertPoint:[event locationInWindow] fromView:nil], [m_view.get() visibleRect]))
```

Comment:
We should store `m_view.get()` in a local to avoid so much ref'ing.

Reply:
This could be simplified further by using the local view instead of the autoreleased getter:

```diff
-    if (m_view.getAutoreleased() == [m_view.get() window].firstResponder && !NSPointInRect([m_view.get() convertPoint:[event locationInWindow] fromView:nil], [m_view.get() visibleRect]))
+    RetainPtr view = m_view.get();
+    if (view == [view window].firstResponder && !NSPointInRect([view convertPoint:[event locationInWindow] fromView:nil], [view visibleRect]))
```